cPP_Lifecycle.adoc

```diff
@@ -0,0 +1,135 @@
+= cPP Lifecycle for the Biometrics Security iTC
+:showtitle:
+:toc:
+:toclevels: 3
+:table-caption: Table
+:imagesdir: images
+:revnumber: 0.1
+:revdate: 2020-08-11
+:xrefstyle: full
+
+:iTC-longname: Biometrics Security
+:iTC-shortname: BIO-iTC
+:iTC-email: isec-itc-bio-info@ipa.go.jp
+:iTC-website: https://biometricitc.github.io/
+:iTC-GitHub: https://github.com/biometricITC/cPP-biometrics
+:iTC-ITname: BIT
+
+== Introduction
+As part of the normal work of the {iTC-shortname}, new iterations of the cPP and SD will be released over time. Ideally, when a new version of the cPP is released, everyone would immediately follow the new version, but of course this is not possible as it takes time to transition to updated requirements.
+
+The purpose of this document is to specify the lifecycle for the cPP versions, providing guidance on when a new version will supersede earlier versions (when the previous version is no longer considered acceptable for use) and how this will be announced to the wider community.
+
+[NOTE]
+====
+While the document talks about cPP revisions, this covers the cPP, SD and any other related documents.
+====
+
+== Revision Classes
+There are three classes of revisions to a cPP that can be produced. The descriptions are not meant to be exhaustive, but to represent the level of changes in the document.
+
+.Revision Classes for a cPP
+[cols=".^1,.^1,.^4",options="header"]
+|===
+
+|Revision Class
+|Version Position
+|Description
+
+|Major
+|**X**.y.zz
+|Full update to the cPP, including requirements changes (addition or removal of requirements)
+
+|Minor
+|x.**Y**.zz
+|Update to the cPP, likely updating existing requirements, but not changing them (i.e. not adding/removing requirements)
+
+|Technical Decision or Errata
+|x.y.**ZZ**
+|Application of Technical Decisions to the cPP (no other changes)
+
+|===
+
+=== Revisions & Branches
+The {itc-shortname} maintains 3 primary branches for the creation of the cPP and all supporting documents. Of these branches, two will be used to maintain the current versions of the documents while the third is used for active work on new updates.
```

Comment:
3 branches, or 2?

Reply:
We have 3, master, develop (or sometimes working) and interpretation. But only master and interpretation are applicable for the "current" version. The develop branch is for working on the next release, not "quick" edits like interpretation, and of course the master is the "official" release.

Reply:
Got it 👍 . Should we add a row in the table for develop/working/etc. ("other")? If so, description might be something like:  _Other branches not explicitly named are used for in-progress development work toward new revisions or TDs. Work in these branches is considered unofficial._

Reply:
@gfiumara Check out the latest paragraph above the table now. I have added some text there to make it clear that only the two branches in the table are the ones that "matter"